src/vl-progress-bar.js

```diff
@@ -0,0 +1,93 @@
+import '/node_modules/@govflanders/vl-ui-progress-bar/dist/js/progress-bar.js';
+import '/node_modules/vl-ui-tooltip/dist/vl-tooltip.js';
+import './vl-progress-bar-item.js';
+
+/**
+ * VlProgressBar
+ * @class
+ * @classdesc Gebruik de progress-bar component om de vooruitgang te laten zien van een proces dat uit verschillende stappen bestaat.
+ * 
+ * @extends VlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-progress-bar/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-progress-bar/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-progress-bar.html|Demo}
+ * 
+ */
+export class VlProgressBar extends VlElement(HTMLElement) {
+    static get _observedChildClassAttributes() {
+        return ['numeric'];
+    }
+
+    constructor() {
+        super(`
+            <style>
+                @import '/src/style.css';
+                @import '/node_modules/vl-ui-tooltip/dist/style.css';
+            </style>
+            <div class="vl-progress-bar"></div>
+        `);
+    }
+
+    connectedCallback() {
+        this._processItems();
+        this._progressBar = new window['progress-bar'](this.constructor);
+    }
+
+    /**
+     * Zet een stap actief al dan niet met focus.
+     * 
+     * @param {number} number - Het nummer van de stap die actief wordt.
+     * @param {boolean} focus - Focus bepaalt of de stap in de progress bar focus krijgt.
+     */
+    updateStep(number, focus) {
+        this._activeItem.active = false;
+        this._getItem(number - 1).active = true;
+        this._progressBar.updateStep(this._shadow, number, focus);
+    }
+
+    get _classPrefix() {
+        return 'vl-progress-bar--'
+    }
+
+    get _items() {
+        return this.querySelectorAll('vl-progress-bar-item');
+    }
+
+    get _activeItem() {
+        return this.querySelector('vl-progress-bar-item[data-vl-active]');
+    }
+
+    _getItem(index) {
+        return this._items[index];
+    }
+
+    _getItemTemplate(item) {
+        const activeClass = item.active ? 'vl-progress-bar__step--active' : '';
+        const ariaCurrent = item.active ? 'aria-current="step"' : '';
+        return this._template(`
+            <div class="vl-progress-bar__step ${activeClass}">
+                <button class="vl-progress-bar__bullet" ${ariaCurrent} aria-label="${item.text}">
+                    <vl-tooltip placement="top">${item.text}</vl-tooltip>
+                    <span class="vl-u-visually-hidden">${item.text}</span>
+                </button>
+            </div>
+        `);
+    }
+
+    _processItems() {
+        this._processActiveItem();
+        const items = [... this._items].map(item => this._getItemTemplate(item));
+        items.forEach(item => this._element.appendChild(item));
+    }
+
+    _processActiveItem() {
+        if (!this._activeItem) {
+            this._getItem(0).active = true;
```

Comment:
_Moet_ er altijd een active step zijn? Kan het ook niet zijn dat ze de steps willen tonen nog voor er bv. aan de wizard begonnen wordt?

Reply:
De progress-bar zal maar getoond worden wanneer de progress begonnen is, wanneer de eerste stap nog niet actief is zal de progress-bar nog niet getoond worden.